Now it actually shows what looks like type bugs or things we need to look at, at least.
https://github.com/prisma/prisma/actions/runs/6640266189/job/18040335862?pr=21614#step:5:24
```
Error: Compilation Error
Error: packages/client/src/runtime/RequestHandler.ts(269,4): error TS7030: Not all code paths return a value.
Error: packages/client/src/runtime/core/engines/library/LibraryEngine.ts(549,74): error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'number'.
Error: packages/client/src/runtime/core/engines/library/LibraryEngine.ts(551,14): error TS18048: 'errorRecord' is possibly 'undefined'.
Error: packages/client/src/runtime/core/extensions/applyQueryExtensions.ts(38,7): error TS2322: Type 'Action' is not assignable to type 'string'.
  Type 'number' is not assignable to type 'string'.
Error: packages/client/src/runtime/core/extensions/applyQueryExtensions.ts(40,7): error TS2578: Unused '@ts-expect-error' directive.
Error: packages/client/src/runtime/core/extensions/applyQueryExtensions.ts(65,79): error TS2345: Argument of type 'Action' is not assignable to parameter of type 'string'.
  Type 'number' is not assignable to type 'string'.
Error: packages/client/src/runtime/core/extensions/applyQueryExtensions.ts(98,7): error TS2322: Type '{ model: string | undefined; operation: Action; args: any; }[]' is not assignable to type 'BatchQuery[]'.
  Type '{ model: string | undefined; operation: Action; args: any; }' is not assignable to type 'BatchQuery'.
    Types of property 'operation' are incompatible.
      Type 'Action' is not assignable to type 'string'.
        Type 'number' is not assignable to type 'string'.
Error: packages/client/src/runtime/core/jsonProtocol/deserializeJsonResponse.ts(31,75): error TS2366: Function lacks ending return statement and return type does not include 'undefined'.
Error: packages/client/src/runtime/core/jsonProtocol/serializeJsonQuery.ts(358,92): error TS2345: Argument of type 'Action' is not assignable to parameter of type 'string'.
  Type 'number' is not assignable to type 'string'.
Error: packages/client/src/runtime/getPrismaClient.ts(805,53): error TS2731: Implicit conversion of a 'symbol' to a 'string' will fail at runtime. Consider wrapping this expression in 'String(...)'.
Error: packages/client/src/runtime/utils/deepCloneArgs.ts(38,43): error TS7030: Not all code paths return a value.
```

--- a/tsconfig.build.regular.json
+++ b/tsconfig.build.regular.json
@@ -2,10 +2,7 @@
   "compilerOptions": {
     "target": "ES2020",
     "module": "commonjs",
-    "lib": [
-      "ES2020",
-      "ES2021.WeakRef"
-    ],
+    "lib": ["ES2020", "ES2021.WeakRef"],
     "esModuleInterop": true,
     "isolatedModules": true,
     "sourceMap": true,

